distribution/docs/src/main/resources/content/_configuring/configuring-from-import-intro-contents.adoc

```diff
@@ -1,14 +1,29 @@
-:title: Configuration Export/Import Intro
+:title: Exporting/Importing Configuration Settings
 :type: configuringImport
 :status: published
 :summary: Export a current ${branding} configuration and use it to restore the system to the same state on a brand new installation.
 :order: 00
 
-The Configuration Export/Import capability allows administrators to export the current ${branding} configuration and use it to restore the same state on a brand new installation or when adding a second node to a highly available deployment.
+==== Exporting/Importing Configuration Settings
+
+The Configuration Export/Import capability allows administrators to export the current ${branding} configuration and use it to restore the same state on a brand new installation or to add a second node to a highly available deployment.
+
+To export the current configuration settings:
+. Run the `migration:export` command from the ${command-console}.
+. Copy the `exported-${project.version}.zip` file from the `exported` (unless specified otherwise when invoking the `migration:export`) directory underneath `<${branding}_HOME>` to a secure place.
+
+To import previously exported configuration settings:
+. Step through the installation process in the exact same location as it was on the exported system. The newly installed ${branding}.
+. Launch the newly installed ${branding}.
+. Copy the previously exported `exported-${project.version}.zip` file to the `exported` (unless specified otherwise when invoking the `migration:import`) directory underneath `<${branding}_HOME>` (you might need to create the directory first).
+. Run the `migration:import` command from the ${command-console}.
+. Finish hardening the system (e.g. file and directory permissions).
 
 [IMPORTANT]
 ====
-* Importing configuration files is only guaranteed to work when importing files from the same ${branding} version.
-  Importing from a different version is not supported.
-* To keep the export/import process simple and consistent, all system configuration files are required to be under the `<${branding}_HOME>` directory.
+* Importing configuration files is only guaranteed to work when importing files from the same ${branding} version and from the same OS.
+  Importing from a different version is not currently supported.
```

Comment:
It might, and often will, work, at least for the most part. Maybe `not guaranteed`?

Reply:
Not anymore. The framework provides support for implementing upgrade logic so when we are ready to add support for upgrade, we simply need to implement what is required. The import will currently fail if attempted.